tools/create-yorkie-app/index.ts

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env node
+
+/* eslint-disable jsdoc/require-jsdoc */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import minimist from 'minimist';
+import prompts from 'prompts';
+import { red, reset } from 'kolorist';
+import { type Framework, FRAMEWORKS } from './FRAMEWORKS';
+
+// Avoids autoconversion to number of the project name by defining that the args
+// non associated with an option ( _ ) needs to be parsed as a string. See https://github.com/vitejs/vite/pull/4606
+const argv = minimist<{
+  t?: string;
+  template?: string;
+}>(process.argv.slice(2), { string: ['_'] });
+const cwd = process.cwd();
+
+const TEMPLATES = FRAMEWORKS.map(
+  (f) => (f.variants && f.variants.map((v) => v.name)) || [f.name],
+).reduce((a, b) => a.concat(b), []);
+
+const renameFiles: Record<string, string | undefined> = {
+  _gitignore: '.gitignore',
+};
+
+const apiKeyMessage = 'You can update your API key in .env';
```

Comment:
Based on the code conventions of yours, this variable would be better named `API_KEY_MESSAGE`.

Reply:
Using upper snake case gave me lint error due to the below rule. Would it be okay to disable it for this file?

```js
// .eslintrc.js
    '@typescript-eslint/naming-convention': [
      'error',
      {
        selector: 'variable',
        format: ['camelCase', 'PascalCase'],
```

Reply:
I didn't know that. then it's okay 🙂